crates/ruff_formatter/shared_traits.rs

```diff
@@ -1,5 +1,3 @@
-#![allow(clippy::all)]
```

Comment:
Lol, this disabled all clippy warnings for some reason. I had to fix some of them after removing it here.